apps/mosaico/src/Mosaico/Footer.purs

```diff
@@ -3,50 +3,60 @@ module Mosaico.Footer
   ) where
 
 import Prelude
+import Data.Foldable (foldMap)
+import Data.Maybe  (Maybe(..))
 import Data.String (Pattern(..), Replacement(..), replaceAll)
-import KSF.Paper (Paper(..), cssName, homepage, paperName)
+import KSF.Paper (Paper(..), homepage, paperName)
 import Mosaico.Ad (openConsentRevocationMessage)
 import React.Basic (JSX)
 import React.Basic.DOM as DOM
-import React.Basic.Events (handler, EventHandler)
 import React.Basic.DOM.Events (preventDefault)
+import React.Basic.Events (handler, EventHandler)
 
 footer :: Paper -> (String -> EventHandler) -> JSX
 footer mosaicoPaper onStaticPageClick =
   DOM.footer
-    { className: "mosaico--footer"
+    { style: DOM.css {"gridArea": "foot"}
```

Comment:
Nice you've used this kind of style too. I've found CSS Grid stuff that doesn't fit comfortably into Tailwind is best put in inline styles like this, to keep everything collocated. No need for a custom CSS class.

Reply:
Yeah, I'm not exactly sure what would be the best thing here... I did also consider writing a small TW plugin for the grid areas, since we are using those quite a lot. Maybe we should just move away from the CSS grid? I don't think it's the right piece of technology for Mosaico anyways.

Reply:
Although how about using `[grid-area:foot]` instead?

Reply:
> Maybe we should just move away from the CSS grid? I don't think it's the right piece of technology for Mosaico anyways.
Pretty much agree. I'm not sure we need it.

Reply:
And also, I've been a bit surprised by how invisible `gap` can end up. A useful concept I'm sure with certain kinds of layouts, but probably not for Mosaico. There's been a few times where I've been wondering why a white space was there, before realising it's because of gap on a parent.